src/Kusto/Kusto/help/Update-AzKustoCluster.md

```diff
@@ -46,9 +46,12 @@ ResourceGroup     : testrg
 Name              : mykustocluster
 Location          : Central US
```

Comment:
Did you run the script to update the help? It seems like you are missing the additional parameter information: https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md.

Reply:
I run the update command.
the output is the same.